wrappers/c/zxing.h

```diff
@@ -0,0 +1,153 @@
+#ifndef _ZXING_C_H
+#define _ZXING_C_H
+
+/*
+ * <stdbool.h>
+ *  bool
+ *
+ * <stdint.h>
+ *  uint8_t
+ */
+#include <stdbool.h>
+#include <stdint.h>
+
+#ifdef __cplusplus
+
+#include <ZXing/ImageView.h>
+#include <ZXing/DecodeHints.h>
+#include <ZXing/Result.h>
+
+typedef ZXing::ImageView zxing_ImageView;
+typedef ZXing::DecodeHints zxing_DecodeHints;
+typedef ZXing::Result zxing_Result;
+typedef ZXing::Results zxing_Results;
+
+extern "C" {
+#else
+
+typedef struct zxing_ImageView zxing_ImageView;
+typedef struct zxing_DecodeHints zxing_DecodeHints;
+typedef struct zxing_Result zxing_Result;
+typedef struct zxing_Results zxing_Results;
+
+#endif
+
+/*
+ * ZXing/ImageView.h
+ */
+
+typedef enum {
+	zxing_ImageFormat_None = 0,
+	zxing_ImageFormat_Lum  = 0x01000000,
+	zxing_ImageFormat_RGB  = 0x03000102,
+	zxing_ImageFormat_BGR  = 0x03020100,
+	zxing_ImageFormat_RGBX = 0x04000102,
+	zxing_ImageFormat_XRGB = 0x04010203,
+	zxing_ImageFormat_BGRX = 0x04020100,
+	zxing_ImageFormat_XBGR = 0x04030201,
+} zxing_ImageFormat;
+
+zxing_ImageView * zxing_ImageView_new (const uint8_t * data, int width, int height, zxing_ImageFormat format, int rowStride, int pixStride);
+void zxing_ImageView_delete (zxing_ImageView * iv);
+
+int zxing_ImageView_width (zxing_ImageView * iv);
```

Comment:
please add a `const` qualifier for the getter functions, like `int zxing_ImageView_width (const zxing_ImageView * iv);`

Reply:
Yup, I thought about that the other day but then forgot.